### ISSUE TITLE
Remove install rules for tests and demos

### ISSUE THREAD

**Issue by ricortiz** [erleben/OpenTissue#21]
Arguably the OT distribution package should only contain the library itself and the CMake machinery find the package and import the OpenTissue target. If users want to run demos then we should instruct them to go over the build process. 

@erleben Thoughts?

**Comment by erleben** [comment]
I agree with you that OT library + CMake is the main parts that make up an OT package.

I am thinking Conan part can be incorporated into the CMake magic workflow. Either guiding users by introspection and meaningful warnings or simply running the Conan install think as a pre-step for the whole CMake thing.

I am also thinking that there are deps on two different levels, those that are used by OT library functions and those that are used by demos/unit-tests. I am in two minds whether they should be treated independently.

**Comment by erleben** [comment]
That is I support nuking the install rules for unit-tests and demos

**Comment by ricortiz** [comment]
Conan can indeed be incorporated into the CMake build system. I have been thinking on a way to do it that will **also** allow folks to build OT aps without conan. 

Ref.

[Launch conan from within cmake.](https://docs.conan.io/en/latest/howtos/cmake_launch.html)

**Comment by ricortiz** [comment]
So here is the dependency breakdown as I see it:
- OpenTissue
> - Boost
> - TetGen
> - Triangle
> - Qhull
> -TinyXML --> this probably should not be a dependency of the core library

unit_test 
> Same as OT

demos
> Same as OT
> GLUT, GLEW and OpenGL

**Comment by erleben** [comment]
We got a bit of devIL as well?

**Comment by erleben** [comment]
The Conan cmake integration seems like a nice solution. It will make it easier for new unexperienced C++ developers to get into OT. Experienced developers can always just dig into the nasty parts, no need to worry about them:-)

**Comment by ricortiz** [comment]
yeah, forgot DevIL...
